server/app/services/question/types/QuestionDefinitionConfig.java

```diff
@@ -0,0 +1,75 @@
+package services.question.types;
+
+import com.google.auto.value.AutoValue;
+import java.time.Instant;
+import java.util.Optional;
+import java.util.OptionalLong;
+import services.LocalizedStrings;
+
+@AutoValue
+public abstract class QuestionDefinitionConfig {
+
+  abstract String name();
+
+  abstract String description();
+
+  // Note: you must check prefixes anytime you are doing a locale lookup
+  // see getQuestionText body comment for explanation.
```

Comment:
suggestion (blocking): This comment is confusing. Maybe make it a Javadoc comment so you can link to the getQuestionText body comment?

Reply:
I didn't write this, it came from here:

https://github.com/civiform/civiform/blob/69750e64b0bdc1d4b6a9b97f70f820fa72a40fa3/server/app/services/question/types/QuestionDefinition.java#L26-L35

But sure - I put it in a Javadoc comment.

Reply:
thanks!